src/components/header/loginButton.tsx

```diff
@@ -71,7 +71,7 @@ const LoginButton = () => {
                     <Button
                       variant="secondary"
                       className="text-white fs-7"
-                      onClick={() => setShowPopover(false)}
+                      onClick={() => { setShowPopover(false); }}
```

Comment:
The additional braces are not required here, they don't make a difference.

Reply:
Ah I think I accidentally left them because I had another instruction afterwards
```suggestion
                      onClick={() => setShowPopover(false}}
```